src/com/company/Main.java

```diff
@@ -1,39 +1,34 @@
 package com.company;
 
-import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.Scanner;
 
 public class Main {
-    //little codelist for field array: 1 -- x; 2 -- o; 3 -- undefined;
-
-    //probably can change this variables
-    static int[][] field = new int[50][50];//field can change size; playable size = size*2;
-    static int limit = 5;//quantity of symbols to win
+    static int[][] field = new int[50][50];
+    static int limit = 5;
 
 
     static boolean winFlag;
     static int x,y;
-    static int currentSymbolCode;
+    static int CurrentSymbol;
```

Comment:
Bad renaming. CurrentSymbolCode better to explain. Because symbol is "o" or "x", code is 1 or 2. In program we work with code of symbols